docs/user/alerting/rule-types/index-threshold.asciidoc

```diff
@@ -1,100 +1,108 @@
-[role="xpack"]
 [[rule-type-index-threshold]]
-=== Index threshold
+== Index threshold
 
 The index threshold rule type runs an {es} query. It aggregates field values from documents, compares them to threshold values, and schedules actions to run when the thresholds are met.
 
 [float]
-==== Create the rule
-
-Fill in the name and optional tags, then select *Index Threshold*.
-
-[float]
-==== Define the conditions
-
-Define properties to detect the condition.
+=== Rule conditions
 
 [role="screenshot"]
-image::user/alerting/images/rule-types-index-threshold-conditions.png[Five clauses define the condition to detect]
+image::user/alerting/images/rule-types-index-threshold-conditions.png[Defining index threshold rule conditions in {kib}]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+When you create an index threshold rule, you must define the conditions for the rule to detect. For example:
 
 Index:: This clause requires an *index or data view* and a *time field* that will be used for the *time window*.
 When:: This clause specifies how the value to be compared to the threshold is calculated. The value is calculated by aggregating a numeric field a the *time window*. The aggregation options are: `count`, `average`, `sum`, `min`, and `max`. When using `count` the document count is used, and an aggregation field is not necessary. 
 Over/Grouped Over:: This clause lets you configure whether the aggregation is applied over all documents, or should be split into groups using a grouping field. If grouping is used, an  <<alerting-concepts-alerts, alert>> will be created for each group when it exceeds the threshold. To limit the number of alerts on high cardinality fields, you must specify the number of groups to check against the threshold. Only the *top* groups are checked.
 Threshold:: This clause defines a threshold value and a comparison operator  (one of `is above`, `is above or equals`, `is below`, `is below or equals`, or `is between`). The result of the aggregation is compared to this threshold. 
 Time window:: This clause determines how far back to search for documents, using the *time field* set in the *index* clause. Generally this value should be to a value higher than the *check every* value, to avoid gaps in detection. 
```

Comment:
Can we add a condition for the optional KQL filter?

Reply:
Agreed, though I'll put it in a separate PR since I want to ponder what other impacts might have arisen from https://github.com/elastic/kibana/issues/66046